app/src/views/private/components/export-sidebar-detail/export-sidebar-detail.vue

```diff
@@ -25,7 +25,7 @@
 
 			<div class="field full">
 				<v-button full-width @click="exportData">
-					{{ t('export_collection', { collection }) }}
+					{{ t('Export ' + collectionName) }}
```

Comment:
This won't work. The string value in the `t()` function matches the translation key under `lang/en-us.yaml`.

```suggestion
					{{ t('export_collection', { collection: collectionName }) }}
```